bubble-wrap-style.yaml

```diff
@@ -2176,7 +2047,7 @@ layers:
                 draw:
                     text-blend-order:
                         priority: 59
-                        visible: global.text_visible_minor_road_e
```

Comment:
Please show before/after images, I remember this being for ~ Tangram JS v0.8 limitations that were later fixed so relying on the v0.15 engine to draw better by default probably makes sense here.